tpcx_bb/cluster_configuration/example-cluster-scheduler.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "Scheduler",
+  "address": "ucx://172.22.1.26:8786",
```

Comment:
We should probably add a comment here explaining what ip the user should use.

Reply:
I believe JSON doesn't support comments, but I think we'll be okay as this is covered in the README

Reply:
Oh, wow, did not know that , Great to learn :-D